parser-typechecker/src/Unison/TermPrinter.hs

```diff
@@ -261,14 +261,7 @@ pretty0
         | otherwise ->
             paren (p >= 11 || isBlock x && p >= 3) $
               fmt S.DelayForceChar (l "'")
-                <> ( case x of
-                       Lets' _ _ -> id
```

Comment:
This case is impossible because of the earlier pattern match on `Lets'` against `x`.

GHC 9.2.3 is smart enough to detect that and issues a warning here.

Reply:
I think we should keep that deleted comment